src/libraries/System.Private.CoreLib/src/System/Half.cs

```diff
@@ -1741,7 +1741,7 @@ static bool INumberBase<Half>.TryConvertToChecked<TOther>(Half value, [NotNullWh
             if (typeof(TOther) == typeof(byte))
             {
                 byte actualResult = checked((byte)value);
-                result = (TOther)(object)actualResult;
+                result = (TOther)(object)actualResult!;
```

Comment:
Why is this operator only needed for the byte conversion?

Reply:
It's not.  This is left over from when I was exploring options with how to handle TOther and I neglected to delete the `!`.  Fixed.